### ISSUE TITLE
High level tracking of Docs Views with HEAP

### ISSUE THREAD

**Issue by noklam** [kedro-org/kedro#2892]
## Description

While I was trying to find out where is our Architecture diagram, I found that our FAQ pages view has gone down signficantly.

![image](https://github.com/kedro-org/kedro/assets/18221871/16e57433-65eb-45a3-9caa-135b1d003d0e)

I try to make sense with the data, but I couldn't make HEAP works. I see consistent pattern across configuration, FAQ, data_catalog etc have a view drop in April while we have highest number of unique user of our docs at the same time.

## Context

I would like to have at least high level tracking of important themes, particularly to help me understand what are the drivers of the view change.

This is hard to do because HEAP is not very flexible (or maybe I don't know enough) and the fact we change our domain from readtodoc -> kedro.org which makes it hard to match the statistics 

- Notebook
- Catalog
- Configuration

This could be useful to understand if we are improving our docs. In addition, it may be useful to track specific features adoption.

**Comment by tynandebold** [comment]
Can you please share the link to that chart? I can help look into this.

**Comment by tynandebold** [comment]
After digging into this I don't think the issue is with the domain changing. Rather, it's the fact that FAQ doesn't exist as a findable top-level item in the navigation nor on some of the pages themselves.

For example, take this page of the docs: `get_started/install.html`. Before we saw the drop in traffic to the FAQ pages, a user could find a link to the FAQs with relative ease in a couple of different places: 

<img width="1506" alt="image" src="https://github.com/kedro-org/kedro/assets/16869061/5fd74b9f-1b6b-4075-9dcc-87c5f46b31d8">


After some recent documentation changes and updates to the navigation and linking, that's no longer the case:

<img width="1480" alt="image" src="https://github.com/kedro-org/kedro/assets/16869061/3557f484-227e-43fc-a425-5d7edf70f57f">


Perhaps we should reconsider removing FAQs from the navigation if that's a page we want to drive traffic to and think more broadly about how important the navigation seems to be for driving traffic to the pages listed there.

cc @stichbury @astrojuanlu

**Comment by astrojuanlu** [comment]
Quick comment, slimming down the FAQs was a deliberate choice https://github.com/kedro-org/kedro/issues/1985

If the data catalog page showed a similar downward pattern though, that's something we should look into

**Comment by tynandebold** [comment]
### Further evidence

In March 2023, 21% of people navigated to the FAQ page from `get_started/install.html`.

<img width="756" alt="image" src="https://github.com/kedro-org/kedro/assets/16869061/028a0605-1fd8-42d1-ada1-17a8cecb689c">

________________

In May, only 6% did the same, a **71% decrease**. 

<img width="748" alt="image" src="https://github.com/kedro-org/kedro/assets/16869061/11464951-9353-4361-85d9-8bb3d7f8f8cf">

**Comment by tynandebold** [comment]
> Quick comment, slimming down the FAQs was a deliberate choice #1985
> 
> If the data catalog page showed a similar downward pattern though, that's something we should look into

Good to know!

**Comment by noklam** [comment]
> Quick comment, slimming down the FAQs was a deliberate choice #1985
> 
> If the data catalog page showed a similar downward pattern though, that's something we should look into

I think I understand what @astrojuanlu is saying here. For example, FAQ page have a question "[How do I specify parameters at run time](https://docs.kedro.org/en/stable/configuration/parameters.html#how-to-specify-parameters-at-runtime)?", if you google it, google will directly you to the pointer https://docs.kedro.org/en/stable/configuration/parameters.html#how-to-specify-parameters-at-runtime, which means the traffic won't go to FAQ but the corresponding page.

However, I don't think this is what happening. The FAQ is now harder to find because it was at the Top Level, but now it's hidden in "Resource". If you search in "FAQ" in Docs, you will not find it because it is "Frequently Asked Question" and RTD search doesn't understand semantic. Google Kedro FAQ will find it because the URL contains "faq" I guess?

Using only 1 filter with Path contains "catalog"
![image](https://github.com/kedro-org/kedro/assets/18221871/48d75eb0-3dca-4ca3-aac6-b32110f43e4e)

Anyway, FAQ is just an example that I found, and I think we can move FAQ back up to the Top level, and maybe alias it as FAQ so it can be searched.

**Comment by stichbury** [comment]
> Perhaps we should reconsider removing FAQs from the navigation if that's a page we want to drive traffic to and think more broadly about how important the navigation seems to be for driving traffic to the pages listed there.

Driving traffic to the FAQ page is only useful if it's of use to the reader. The reason we slimmed down the page was because it wasn't -- it had marketing content about Kedro (still useful, but not when someone is _learning_ how to use it rather than learning about it). I'd be interested to see the dwell time on the page from the traffic prior to the changes and compare to after, since I suspect a lot of people bounced off it swiftly. 

It would also be useful @tynandebold to see if traffic to the website page rose from organic traffic when we did the FAQs move.

Let's think about our FAQs in terms of the ways people encounter and use the page:

1. They have a *specific* technical question "how can I specify a parameter to `kedro run`?". They'll type that into search and navigate via links, whether on google or internal site search. Prior to our changes, when FAQs was a top-level page, they would not have hit the page from search often because it wasn't a technical query we covered in FAQs (ours were a scattered collection of stuff we couldn't quite work out where it went). Now they have a better chance of it being there. It's still low as that page is limited, but there are distinct queries on the page (and the rest of the site content is gradually changing in terms of SEO opportunity to answer organic technical search). We still need to work out how to provide more FAQ content/knowledgebase content but I'm fairly sure a page in the docs is *not* the way forward for this, given we have Slack archives, GenAI opportunities and not enough people to wizard and maintain documented FAQs.
2. When people aren't using search to find queries, they may encounter and use the page through browsing, either following a link elsewhere or browsing for it, or seeing it on nav. You can't really target content for this use case, but you can make it clear what the page is for, so they can either get something from it there and then, or use it in future. A long page of text covering random questions isn't doing that. 

Traffic for use case 2 is what we have reduced, and TBH that's OK. It wasn't particularly useful for them except as a "Oh look, this page is here and tells me about data layers" as it was. We can and should make it more discoverable if we want more traffic, but we're not here to move people about the site but to serve their needs as readers.

## TL;DR

@tynandebold's data shows that the drop is because of the change in location & linking. So...

* We do need to make the page easier to find for use case 2 (random discovery) and retitle it as "FAQs"
* We don't need to revise the content back to what it was

**Comment by stichbury** [comment]
I made some revisions to the FAQs and other pages affected by FAQs (plus some housekeeping for google indexing of slack archive) in #2909 and I'm closing this issue for now. Let's keep an 👁️ on the data for the FAQs page though.
